testdata/no_source_names.designspace

```diff
@@ -0,0 +1,35 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<designspace format="4.1">
+  <axes>
+    <axis tag="wght" name="Weight" minimum="400" maximum="700" default="400"/>
+  </axes>
+  <sources>
+    <source filename="TestFamily-Regular.ufo" familyname="Test Family" stylename="Regular">
+            <location>
+        <dimension name="Weight" xvalue="400"/>
+      </location>
+    </source>
+    <source filename="TestFamily-Bold.ufo" familyname="Test Family" stylename="Bold">
+      <location>
+        <dimension name="Weight" xvalue="700"/>
+      </location>
+    </source>
+  </sources>
```

Comment:
Optional: make this the most minimal source thatis value, e.g. no familyname or any other optional attribute